soletta/simple_type_c.go

```diff
@@ -0,0 +1,466 @@
+package soletta
+
+/*
+#include <ctype.h>
+#include <stdlib.h>
+#include <stdarg.h>
+#include <sol-log.h>
+#include <sol-flow.h>
+
+#include "simple_type_c.h"
+
+#define streq(a, b) (strcmp((a), (b)) == 0)
+
+struct sol_flow_node {
+    const struct sol_flow_node_type *type;
+    struct sol_flow_node *parent;
+    char *id;
+
+    void *parent_data;
+
+    void *data[];
+};
```

Comment:
It seems you don't need to handle the internal fields of this struct directly, so please do not re-define if it's not necessary (you can use the symbol direct from sol-flow.h)